features/enable_fips_cloud.feature

```diff
@@ -130,192 +64,50 @@ Feature: FIPS enablement in cloud based machines
         And I verify that `openssh-server-hmac` installed version matches regexp `fips`
         And I verify that `openssh-client-hmac` installed version matches regexp `fips`
         And I verify that `strongswan-hmac` installed version matches regexp `fips`
-        When I run `apt-mark unhold openssh-client openssh-server strongswan` with sudo
-        Then I will see the following on stdout:
-        """
-        openssh-client was already not hold.
-        openssh-server was already not hold.
-        strongswan was already not hold.
-        """
-        When I run `ua status --all` with sudo
-        Then stdout matches regexp:
-            """
-            <fips-service> +yes                disabled
-            """
 
         Examples: ubuntu release
-           | release | fips-name    | fips-service |fips-apt-source                                |
-           | bionic  | FIPS         | fips         |https://esm.ubuntu.com/fips/ubuntu bionic/main |
+           | release | fips-apt-source                                |
+           | bionic  | https://esm.ubuntu.com/fips/ubuntu bionic/main |
```

Comment:
Can we just add a focal line here instead of having a separate scenario?
```suggestion
           | bionic  | https://esm.ubuntu.com/fips/ubuntu bionic/main |
           | focal  | https://esm.ubuntu.com/fips/ubuntu focal/main |
```

Reply:
The problem is that we don't have the openssh fips packages on focal. That's why it is in another scenario

Reply:
Ahhhh right, thanks I missed that